wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/solenoid/Robot.java

```diff
@@ -4,11 +4,23 @@
 
 package edu.wpi.first.wpilibj.examples.solenoid;
 
+// Using "import static an.enum.or.constants.inner.class.*;" helps reduce verbosity
```

Comment:
I don't like the static import for an example, since other parts of WPILib use kForward and kReverse (Relays). It's not worth the confusion for reduced verbosity

Reply:
We recommend it in the docs.

Reply:
Well, I don't like it there either

Reply:
@sciencewhiz do you want to remove that section from the docs?

Reply:
Yes, I would prefer that